doc/source/howtos/encrypt-traffic.rst

```diff
@@ -236,4 +315,46 @@ assumig there are two nodes in the cluster:
 
       wsrep_provider_options=socket.ssl=yes;socket.ssl_ca=/path/to/upgrade-ca.pem;socket.ssl_cert=/path/to/new-cert.pem;socket.ssl_key=/path/to/new-key.pem
 
+.. _ssl-auto-conf:
+
+SSL Automatic Configuration
+===========================
+
+|PXC| includes the :variable:`pxc-encrypt-cluster-traffic` variable
+that enables automatic configuration of SSL encrytion.
+By default, it is disabled, meaning that you need to configure SSL manually
+if you want to encrypt SST and IST traffic, and internal communication.
+
+This variable cannot be changed on the command line during runtime.
+To enable automatic configuration of SSL encryption,
+set ``pxc-encrypt-cluster-traffic=ON`` in the :file:`my.cnf` file.
+
+When you enable automatic configuration of SSL encryption,
+it looks for necessary SSL files generated by MySQL in
+the ``ssl-ca``, ``ssl-cert``, and ``ssl-key`` options under ``[mysqld]``.
+If these options are not set, it then looks in the data directory for
+:file:`ca.pem`, :file:`server-cert.pem`, and :file:`server-key.pem` files.
+
+.. note:: The ``[sst]`` section is not searched.
+
+If all three files are found,
+they are used to configure Galera SSL and SST configuration with ``encrypt=4``.
+Other modes are deprecated and will be overriden
+if automatic configuration of SSL encryption is enabled.
+If any of the files are missing, a fatal error is generated.
+
+The following settings are applied or overridden::
```

Comment:
Maybe this should be written as:
The following settings will be applied and will override any existing settings.